README.md

```diff
@@ -109,48 +112,47 @@ If prompted for a password, input your currently logged-in user's password to pe
 ```graphql
   type User {
     id: ID!
-    Email: String!
-    Password: String!
-    FirstName: String!
-    LastName: String!
-    Gender: String
-    Address: String!
-    Latitude: Float!
-    Longitude: Float!
-    Photo: String
-    Events_Owned: [Event]!
-    Events_Invited: [Event]!
-    Events_Attending: [Event]!
```

Comment:
Is there no way to show which Events a given User has been invited to or is attending?

Reply:
I can definitely try to bring those arrays back. Let me work on it for bit and push up another changes.